### ISSUE TITLE
please add a feature to get all mention list while we type 'Ctrl+Space'

### ISSUE THREAD

**Issue by Neethukp93** [dmacfarlane/angular-mentions#78]
I want to get all the mention list whenever I am typing Ctrl+Space . It will be helpful if you can integrate that feature .

**Comment by Neethukp93** [comment]
yes...that is what I exactly need. when i press Ctrl+space , mention list need to be displayed in the "textarea" without any prefix character.

**Comment by Alec-Aldrine-Lakra** [comment]
```
     const event = new KeyboardEvent("keydown", { key : `@`, code : `Digit2`});
```

Every time you hit Ctrl+Space create a keyboard event

**Comment by steve-heine** [comment]
@Alec-Aldrine-Lakra  this will show the items list, but will not let you interact with it (arrow keys or click)

**Comment by steve-heine** [comment]
the problem seems to occur because pos === this.startPos, but the code requires pos > this.startPos

**Comment by RomanTseduliak** [comment]
Is it possible to check for the presence of a previous character? so that the trigger does not work when it is called in the middle of the word?
